public/main.js

```diff
@@ -3,54 +3,36 @@ const randomNumber = Math.floor(Math.random() * (100) + 1)
 let counter = 0
 
 const guessNumber = (input) => {
-  let inputValue = input.value
-  console.log(randomNumber);
-  if (isNaN(inputValue) || inputValue < 1 || inputValue > 100) {
-    alert('Must be a number between 1 and 100')
-  } else if (inputValue == randomNumber) {
-    alert('You Win!')
-  } else if (inputValue > randomNumber) {
-    alert('Guess Lower')
-    counter++
-  } else if (inputValue < randomNumber && inputValue >= 0) {
-    alert('Guess Higher')
-    counter++
-  }
-  document.getElementById("guesses").innerHTML = "Guesses so far: " + counter
+    let inputValue = input.value
+    console.log(randomNumber);
```

Comment:
Is this just for your testing purposes? If someone opens the devtools console while running this you can see the answer. I would remove this from the PR.